### ISSUE TITLE
Error when running example

### ISSUE THREAD

**Issue by dfucci** [UPHF/RTj#3]
After running the example presented in the README, as follows
`
java -cp /Users/dfu/Projects/RTj/target/rtj.jar fr.inria.jtanre.rt.RtMain                                                                                                                            
-location  /Users/dfu/Projects/RTj/examples/rt-project 
-out   ./ouput
`

I got the following error(s):
```
[INFO] 13:57:43,175 fr.inria.astor.core.setup.ProjectConfiguration:282 - Version of the JVM used: 1.8.0_242  
[INFO] 13:57:43,178 main:381 - Java version of the JDK used to run tests: 1.8.0_242
[INFO] 13:57:43,178 main:382 - The compliance of the JVM is:  8
[DEBUG] 13:57:43,179 main:678 - Determining project properties from /Users/dfu/Projects/RTj/examples/rt-project
/Users/dfu/bin/mvn false
/usr/local/bin/mvn true
[DEBUG] 13:57:45,163 main:699 - [ERROR] Error resolving version for plugin 'com.github.tdurieux:project-config-maven-plugin' from the repositories [local (/Users/dfu/.m2/repository), central (https://repo.maven.apache.org/maven2)]: Plugin not found in any plugin repository -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginVersionResolutionException

Exception in thread "main" com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 10 path $
	at com.google.gson.JsonParser.parse(JsonParser.java:65)
	at com.google.gson.JsonParser.parse(JsonParser.java:45)
	at fr.inria.main.AbstractMain.executeAutoConfigure(AbstractMain.java:700)
	at fr.inria.main.AbstractMain.processArguments(AbstractMain.java:661)
	at fr.inria.main.evolution.AstorMain.execute(AstorMain.java:204)
	at fr.inria.jtanre.rt.RtMain.main(RtMain.java:93)
Caused by: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 10 path $
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1568)
	at com.google.gson.stream.JsonReader.checkLenient(JsonReader.java:1409)
	at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:542)
	at com.google.gson.stream.JsonReader.peek(JsonReader.java:425)
	at com.google.gson.JsonParser.parse(JsonParser.java:60)
	... 5 more
```
The first part of the error seems to be related to the [project-info-maven-plugin](https://github.com/tdurieux/project-info-maven-plugin). However, I have installed the plugin, as follows:
`mvn org.apache.maven.plugins:maven-dependency-plugin:3.1.1:get -DremoteRepositories=https://tdurieux.github.io/maven-repository/snapshots/ -Dartifact=com.github.tdurieux:project-config-maven-plugin:1.0-SNAPSHOT`

**Comment by martinezmatias** [comment]
Hi @dfucci 
Thanks for reporting the problem.
This maven plugging is used to get the information of the project under analysis. 
Its usage is optional (ie., we can pass that info via command line ), I will update and improve the documentation. 
W.r.t.  project-info-maven-plugin, I had some inconvenient when I installed as you mentioned, So I proceed to clone the project and then execute a maven install.
